src/lib.rs

```diff
@@ -14,11 +22,58 @@
 //!
 //! # Events handling
 //!
-//! Once a window has been created, you can handle the events that it generates. There are two ways
-//! to do so: with `poll_events` or with `wait_events`. The former returns an iterator that ends
-//! when no event is available, and the latter returns an iterator that blocks and waits for events
-//! if none is available. Depending on which kind of program you're writing, you usually choose
-//! one or the other.
+//! Once a window has been created, it will *generate events*. For example whenever the user moves
+//! the window, resizes the window, moves the mouse, etc. an event is generated.
```

Comment:
Just a thought, would it be the proper place to add a note that on some platforms, no events will be generated before something is drawn on the window (this is the case of wayland)? This can totally be done in a followup PR btw, it's just that reading this here made me think of it.